package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "storm-jsonrpc",
+  "version": "0.0.1",
+  "author": "Sreelal Nampoothiri <sreelalhn@gmail.com>",
+  "scripts": {
+    "start":"node -r esm src/storm_jsonrpc.js",	 
+    "lint": "eslint '**/*.js'",
+    "lint:fix": "eslint '**/*.js' --fix",
+    "postinstall": "(test -e testcases/Storm-Testcases || git clone https://github.com/WebPlatformForEmbedded/Storm-Testcases.git testcases/Storm-Testcases)"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
pre commit hook configured with eslint (nice 👍 ), but eslint configuration itself is missing.

Feel free to reuse the config ('.eslintrc.js' and `.eslintignore`) from the Storm-CLI repo (https://github.com/WebPlatformForEmbedded/Storm-CLI)

Reply:
Thanks. Added eslint configuration. For now we have added minimal configuration , later we will enhance the same.